src/index.ts

```diff
@@ -225,24 +227,36 @@ function main() {
                     }
                 })
                 .then(({appId, domainName, objectId, servicePrincipalId, servicePrincipalSecret}) => {
+                    cachedAuthResponse.options.tokenAudience = null;
+                    answers.deploymentSku = program.sku;
+                    answers.runtime = program.runtime;
+                    if (program.versionOverride === 'master') {
```

Comment:
other than 'master' what are the other valid input values that are expected ? Should we make this an enum instead ?

Reply:
I actually only want master branch to be the valid value if it is not using a published version.